### ISSUE TITLE
Axe core throws exception for wcag21a tag

### ISSUE THREAD

**Issue by lpriyasiva** [dequelabs/axe-core#1391]
Expectation: Axe 3.1.2 should ignore wcag21a tag
Actual: Axe 3.1.2 throws an exception if wcag21a tag is included in the ruleset

Motivation: In comply, we have rules in a standard template file. We are unable to add wcag21a as a tag in case new standards get added for it.

<pre><code>
axe-core version: 3.1.2
axe-webdriver, extension or other integration version: X.Y.Z

Browser and Assistive Technology versions

For Tooling issues:
- Node version: XX  
- Platform:  
</code></pre>

**Comment by WilcoFiers** [comment]
Howdy, thank you for reporting. There are currently no wcag21a rules, so that is why axe-core is throwing the error. I talked to @dylanb about this a few weeks ago and we concluded this erroring out when it finds no matching tags is probably a bit too aggressive. We'll make sure to have this work a little more gracefully.

**Comment by straker** [comment]
I believed this was fixed in https://github.com/dequelabs/axe-core/pull/1290

**Comment by chandana7393** [comment]
There is no wcag21a in Attest ruleset and also in comply Testing standards there is  consistency over everywhere.Istead of wcag21a there are using wcag21aa.
![ubuntu_regression_comply_server](https://user-images.githubusercontent.com/45285854/59420789-20e1a300-8deb-11e9-9b13-f06f71b3bce1.PNG)
![Attest_ruleset](https://user-images.githubusercontent.com/45285854/59420987-7a49d200-8deb-11e9-865c-15eba23f280b.PNG)
**Tested Environment:**
Attest - 2.5.0.15720v
Comply server Url  - https://comply-qa-centos-core.dequelabs.com
Comply version - v6.3.1.5792
